vertica_python/vertica/cursor.py

```diff
@@ -9,15 +9,12 @@
 
 
 class Cursor(object):
-
-    def __init__(self, connection, cursor_type=None, row_handler=None):
```

Comment:
With iterate, passing the handler to the ctor is not needed. Having a handler embedded in the cursor also created several inconsistency.